section/storage.html

```diff
@@ -1,23 +1,167 @@
 <section id="storage">
     <h2>Storage</h2>
-
-    <pre class="idl">
-        dictionary Storage {
-            required boolean connected; // default `false`
-            required boolean enabled; // whether continuous payments are enabled
-
-            required DOMString privateKey; // }
-            required DOMString publicKey;  // } should be stored in a secure storage
-            required DOMString kid;        // }
-
-            required WalletAddressDetails walletAddress;
-            required DOMString rateOfPay;
-            required DOMString maxRateOfPay;
-
-            // TBD
-            // required object grantDetails;
-            // required object exceptionList;
-        };
-    </pre>
-
+    
+    <p>This section defines the browser storage requirements for persisting Web Monetization user data and session state.</p>
+    
+    <section data-dfn-for="Profile">
+        <h3>Profile Storage</h3>
+        
+        <p>The browser MUST securely store an instance of {{Profile}} to maintain Web Monetization state across page loads and browser restarts. This storage MUST be:</p>
+        
+        <ul>
+            <li>Persistent across browser sessions</li>
+            <li>Encrypted when stored on disk</li>
+        </ul>
+        
+        <p class="note">
+            Implementations SHOULD use secure storage mechanisms such as the browser's credential store, encrypted local storage, or platform-specific secure storage APIs. Private keys and access tokens are particularly sensitive and require the highest level of protection.
+        </p>
+        
+        <section data-dfn-for="Profile">
+            <h4>Profile</h4>
+            
+            <p>The {{Profile}} dictionary represents the complete user state for Web Monetization, containing wallet information, authentication credentials, active grants, and payment preferences.</p>
+            
+            <pre class="idl">
+            dictionary Profile {
```

Comment:
Nit: I'm open to calling Profile something else, but I don't have any good suggestions. Maybe something more explicit like `WebMonetizationPeristentStorage`?

Reply:
Makes sense on rest of the dictionaries not having the prefix.

I am more confused by "Profile". Storage is sensible but can get confused with the [Storage interface](https://html.spec.whatwg.org/multipage/webstorage.html#storage-2) as you said. Perhaps PersistentData?

Reply:
I’ll keep using Storage for the time being until we find a better name. PersistentData could be an option, though it doesn’t feel much more descriptive than Storage.